### DISCUSSION TITLE
Support for AKS?

### DISCUSSION THREAD

**Discussion by dtseiler** [zalando/postgres-operator#1427] [Q&A]
Does your postgres-operator support Microsoft's Azure Kubernetes Service (AKS)? I tried to deploy it there but ran into problems and wondering if I should expect it to work.

**Comment by steav** [comment]
Yes, we have been running it smoothly on AKS since a year in production + multiple namespaces.

**Reply by dtseiler** [reply]
Hmm ok I'll have to take another look at it. I'm not a kubernetes wizard either so I'm sure there are some pieces that I'm missing on my end.
